libs/libdaemon/src/BaseDaemon.cpp

```diff
@@ -436,19 +436,21 @@ static void terminate_handler()
 }
 
 
-static std::string createDirectory(const std::string & _path)
+static std::string createDirectory(const std::string & _path, bool strip_file = 1)
```

Comment:
This is ancient code. Why not to throw away and use `Poco::File::createDirectories instead`?